jobs/cronjob-ldap-group-sync.yml

```diff
@@ -169,6 +210,40 @@ parameters:
     description: "LDAP Filter to use when deciding which groups to sync into OpenShift"
     required: true
     value: "(&(objectclass=ipausergroup)(memberOf=cn=openshift-users,cn=groups,cn=accounts,dc=myorg,dc=example,dc=com))"
+  - name: "LDAP_GROUP_NAME_ATTRIBUTES"
+    displayName: "Group name attributes"
+    description: "The attribute list to use to discover the name for the group."
+    required: true
+    value: >-
+      ["cn"]
+  - name: "LDAP_GROUP_MEMBERSHIP_ATTRIBUTES"
+    displayName: "Group membership attributes"
+    value: >-
+      ["member"]
+  - name: "LDAP_GROUP_UID_ATTRIBUTE"
+    displayName: "Group UID Attribute"
+    description: "The attribute that uniquely identifies a group on the LDAP server."
+    required: true
+    value: "ipaUniqueID"
+  - name: "LDAP_INSECURE"
+    displayName: "LDAP Insecure"
+    description: "Disable TLS negotiation for LDAP. If set to true, then LDAP_CA_CERT_PATH must be set to empty string."
+    required: true
+    value: "false"
```

Comment:
To stay backwards compatible, can we please default this to `true`? Also, I'd recommend moving this parameter up, next to the certs params.